app/services/AccountingPeriodService.scala

```diff
@@ -46,4 +46,26 @@ class AccountingPeriodService @Inject()(currentDateProvider: CurrentDateProvider
   }
 
   def currentTaxYear: Int = AccountingPeriodUtil.getTaxEndYear(currentDateProvider.getCurrentDate)
+
+  object agentUpdateDates {
```

Comment:
```suggestion
  object AgentUpdateDates {
```
Objects tend to start uppercase